web/static/js/components/respondents/RespondentIndex.jsx

```diff
@@ -109,6 +109,18 @@ class RespondentIndex extends Component<Props, State> {
     this.props.actions.sortRespondentsBy(projectId, surveyId, name)
   }
 
+  getModeAttempts() {
+    const { survey, sortBy, sortAsc, t } = this.props
+    let surveyModes = survey.mode[0]
+    let attemptsHeader = []
+    surveyModes.forEach(function(mode) {
+      let modeTitle = mode.charAt(0).toUpperCase() + mode.slice(1) + ' ' + 'Attempts'
```

Comment:
```suggestion
      const capitalize = str => str.charAt(0).toUpperCase() + str.slice(1)
      let modeTitle = capitalize(mode) + ' ' + 'Attempts'
```

or in separate function if is used in other place